docs/excel/custom-functions-best-practices.md

```diff
@@ -94,7 +94,7 @@ The following sample shows the JSON metadata that corresponds to the functions d
 
 ## Additional considerations
 
-Avoid accessing the Document Object Model (DOM) directly or indirectly (for example, using jQuery) from your custom function. On Excel for Windows, where custom functions use the [JavaScript runtime](custom-functions-runtime.md), custom functions cannot access the DOM.
+Avoid accessing the Document Object Model (DOM) directly or indirectly (for example, using jQuery) from your custom function. In Excel on Windows, where custom functions use the [JavaScript runtime](custom-functions-runtime.md), custom functions cannot access the DOM.
```

Comment:
Thanks for fixing these, this is one of those I'll have to add to my list - I didn't realize it's FOR mac/ipad but ON windows